articles/liquid/liquid.md

```diff
@@ -0,0 +1,136 @@
+
+## **Overview**
+
+Liquid Network is a sidechain-based settlement network that functions as a Bitcoin Layer-2 solution. It interconnects global cryptocurrency exchanges and institutions, facilitating rapid and confidential Bitcoin transactions along with the issuance of digital assets.
+
+Liquid aims to address some of the limitations of the Bitcoin network by providing faster transaction settlements and enhanced privacy, all while maintaining robust security. It enables the issuance of various types of digital assets which opens new avenues for trading and asset management on its platform.
+
+The network was officially launched in 2018 by Blockstream, and it operates as an implementation of Elements, a sidechain-capable blockchain platform. 
+
+![l1](./img/l1.png)
+
+## **Protocol Quick Scan**
+
+The Liquid Network was specifically designed as a sidechain to Bitcoin that aims to address the scalability and privacy limitations of its parent chain. This section provides a concise overview of how the Liquid Network operates, its unique features, and the technical innovations it introduces to enhance transaction capabilities on Bitcoin.
+
+### **Operational Mechanics**
+
+At its core, the Liquid Network facilitates faster and more private transactions through a sidechain mechanism linked directly to the Bitcoin blockchain. Users can transfer Bitcoin into the Liquid Network using a two-way peg, which converts BTC to Liquid Bitcoin (L-BTC) at a 1:1 ratio. This conversion enables the use of L-BTC across Liquid’s ecosystem with all transactions benefiting from the sidechain’s enhanced features like rapid settlement times and increased privacy.
+
+### **Key Differentiators**
+
+**Federated Consensus**: Unlike Bitcoin’s decentralized proof-of-work system, Liquid utilizes a federated consensus model managed by a consortium of trusted functionaries. These functionaries are responsible for maintaining the network’s integrity and security. This model allows for consistent block times and immediate transaction finality, reducing the typical wait times associated with Bitcoin’s block confirmations.
+
+**Confidential Transactions**: Liquid introduces privacy as a core component of its architecture. Through Confidential Transactions, transaction amounts and asset types are obscured, providing privacy for financial transactions while still maintaining the ability to audit and verify transfers securely. This feature is particularly appealing to institutions and individuals seeking privacy without sacrificing transparency.
+
+### **Technical Innovations**
+
+**Liquid’s Two-Way Peg**: The pegging mechanism used by Liquid allows for secure and verifiable movement of Bitcoin between the Bitcoin blockchain and the Liquid sidechain. This peg is crucial for maintaining a trustless exchange environment where users can move assets across blockchains without reliance on intermediary transfer services.
```

Comment:
The technical innovation of two-way pegging cannot be inferred from the context as it is one of the most general mechanism of sidechains. In this case, tt should be reflected why it can be considered as a innovation